pnpm-lock.yaml

```diff
@@ -53,8 +53,6 @@ overrides:
   yallist: ^5.0.0
   yargs: ^17.7.2
 
-pnpmfileChecksum: sha256-zddoD5DO4pDXt1woGSkWiGYuIzHsyOAVI3HXb7mgvsM=
```

Comment:
This issue is kind of annoying... Do we have any ideas on how to solve this yet?

Reply:
Not yet ...